internal/website/content/howto/runtimervar/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Runtime Configuration"
+date: 2019-06-18T14:51:12-07:00
+showInSidenav: true
+---
+
+Working with runtime variables using the Go CDK takes two steps:
+
+1. Open the variable with the `runtimevar` provider of your choice.
+2. As many times as needed, use the `Latest` or `Watch` methods to fetch the
+   value of the variable.
+
+## Constructors versus URL openers
+
+The easiest way to open a variable is using [`runtimevar.OpenVariable`][] and a
+URL pointing to the variable. See [Concepts: URLs][] for more
+details. If you need fine-grained control over the connection settings, you can
+call the constructor function in the driver package directly (like
+`awsparamstore.OpenVariable`). This guide shows how to use both forms for
+each storage provider.
+
+[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+
+[`runtimevar.OpenVariable`]:
+https://godoc.org/gocloud.dev/runtimevar#OpenVariable
+
+## GCP Runtime Configurator {#gcp-url}
+
+To open a variable stored in [GCP Runtime Configurator][] via a URL, you can use
+the `runtimevar.OpenVariable` function as follows. It uses the `string`
+[decoder][].
```

Comment:
Decoders are not introduced yet, I would add a section on them above.

Reply:
Right, so I linked "decoder" to the doc. I think it's sufficient at this point, since the reference documentation (linked) describes decoders fairly well

Reply:
IMHO it would be good to have a section on Decoders in here; this introduction with just a link to godoc is too abrupt. We should explain that you can tell `runtimevar` to interpret your config as JSON, a string, or `[]byte`, and that that determines the resulting type in the `Snapshot`. It's not obvious.

Reply:
Beefed it up a bit, without a separate section. Should be much less abrupt now, while still linking to the reference doc now.

The reason I'm objecting to a separate section is that it (1) breaks the flow of just showing a simple end-to-end example, and even (2) if placed in the end, would repeat a lot of what the linked reference describes. I think the current text is a reasonable compromise. The user gets a quick idea of what decoders mean and a link for more info if needed.